project_legal_management/views/project_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) -->
+<odoo>
+    <record id="edit_project" model="ir.ui.view">
+        <field name="name">project_project_form_page_other_info</field>
```

Comment:
Same as before, you should include the name of the module to clearly identity it in the view hierarchy